xml/cha_introduction.xml

```diff
@@ -296,4 +296,52 @@
   </important>
  </sect1>
 
+ <sect1 xml:id="sec-intro-support">
+  <title>Support</title>
+  <para>
+   Regardless of the public cloud you use and the plan you choose, &suse; has
+   you covered with support. Details however differ:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     PAYG instances include support through the cloud provider which in turn is
+     supported by &suse;. The CSP provides 1st and 2nd level, &suse; will
+     provide level 3 if necessary.
+    </para>
+    <para>
+     This means that you <emphasis>always contact your cloud service provider
+     first</emphasis>. They will assist you with your problem. If no resolution
+     can be foud, they will open a support ticket for you with &suse;.
+    </para>
+    <important>
+     <title>&azure; basic images</title>
+     <para>
+      &azure; also offers a basic PAYG image that only includes updates.
+      Instances created from this image are not eligible for support. The only
+      way to get support is by reproducing the problem on an identical instance
+      created from a supported PAYG image.
```

Comment:
IMHO we do not want to highlight this route. We do not really want customers to primarily run basic and then set up a "fully supported" instance when they need/want to reproduce an issue. IMHO highlighting this in the documentation encourages the gaming of the system, We should not do this, IMHO.

Reply:
Agreed, that's why I asked in my 1st comment. Removed.